openassessment/templates/openassessmentblock/oa_base.html

```diff
@@ -3,7 +3,7 @@
 <div class="wrapper wrapper--xblock wrapper--openassessment theme--basic">
     <div class="openassessment problem" id="openassessment">
         <div class="wrapper--grid">
-            <h2 class="openassessment__title problem__header">{{ title }}</h2>
+            <h2 class="openassessment__title problem__header">{% trans title %}</h2>
```

Comment:
@nedbat Yes, for some reason when I added the translation into openassessmentblock.py, it never got added to the list of things to translate, and I was getting some quality errors about _ not being used.